custom_components/better_thermostat/climate.py

```diff
@@ -305,23 +315,23 @@ def _async_startup(*_):
 					if self.night_temp and isinstance(self.night_temp, numbers.Number):
 						_night_temp = float(self.night_temp)
 						# if the night temperature is lower than min_temp, set it to min_temp
-						if _night_temp < self.min_temp:
+						if _night_temp < self._TRV_min_temp:
```

Comment:
Nope. I think we break user expectations if we can set a night temperature below our minimum configurable temperature. 

We should just log this as a (configuration) error and move on.